src/python/aqueduct_executor/operators/function_executor/extract_function.py

```diff
@@ -52,7 +52,7 @@ def extract_function(storage: Storage, spec: FunctionSpec) -> None:
     """
     fn_path = spec.function_extract_path
     if not os.path.exists(fn_path):
-        os.mkdir(fn_path)
+        os.makedirs(fn_path)
```

Comment:
Is there a difference between the two?

Reply:
Apparently os.mkdir doesn't work on lambda for some reason, the suggestion is to use os.makedirs